cdap-docs/developers-manual/source/metadata/index.rst

```diff
@@ -1,23 +1,31 @@
 .. meta::
     :author: Cask Data, Inc.
-    :copyright: Copyright © 2015 Cask Data, Inc.
+    :copyright: Copyright © 2015-2017 Cask Data, Inc.
 
-.. _metadata-lineage:
+:hide-toc: true
+
+.. _metadata:
+
+========
+Metadata
+========
+
+.. toctree::
+   :maxdepth: 1
+   
+    CDAP Metadata UI <metadata-ui>
```

Comment:
Let's discuss how to handle this, as we have run into a current limitation of Sphinx. What's happening is that all rst files need to appear at least once in an table of contents somewhere, so that they can be found and linked together. They are either in a defined `table-of-contents.rst` or in another file, always as part of a ``.. toctree::`` directive. The problem is that the directive always gets inserted first into the sidebar. I don't have control over it, though I guess I could do something about it if it was really required. It would necessitate digging into the Sphinx internals and figuring out how to change the order.

How I have handled it is to instead do a top-level page that has *no* section breaks in it, that is an "overview" page, and then have all pages below it appear as links in the left sidebar.

See http://builds.cask.co/artifact/CDAP-DQB256/shared/build-8/Docs-HTML/4.1.0-SNAPSHOT/en/developers-manual/data-exploration/index.html for an example.